azure-devops/azext_devops/dev/team/invoke.py

```diff
@@ -0,0 +1,148 @@
+# pylint: skip-file
+# flake8: noqa
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from knack.util import CLIError
+
+from azext_devops.dev.common.services import (resolve_instance,
+                                              get_vss_connection)  
+
+from azext_devops.vstsCompressed.vss_client import VssClient
+
+
+logger = get_logger(__name__)
+
+def invoke(area=None, resource=None,
+           route_parameters=None,
+           query_parameters=None,
+           api_version='4.1',
+           http_method='GET',
+           in_file=None,
+           media_type='application/json',
+           accept_media_type='application/json',
+           out_file=None,
+           organization=None, detect=None):
+    """ This command will invoke request for any DevOps area and resource
+    """
+
+    logger.info('route_parameter received is %s', route_parameters)
+    version = apiVersionToFloat(api_version)
+
+    organization = resolve_instance(detect=detect, organization=organization)
+    connection = get_vss_connection(organization)
+
+    request_body = None
+    if in_file:
+        with open(in_file) as f:
+            import json
+            request_body = json.load(f)
+
+    resource_areas = connection._get_resource_areas(force=True)
+
+    if(not area and not resource):
+        service_list = []
+
+        for x in resource_areas:
+            if x.location_url not in service_list:
+                service_list.append(x.location_url)
+
+        resouce_locations = []
+
+        for x in service_list:
+            try:
+                logger.info('trying to get locations from ' + x)
+                clientMock =  VssClient(x, connection._creds)
+                resrouce_location_on_this_service = clientMock._get_resource_locations(all_host_types=True)
```

Comment:
Typos : **resouce**_locations and **resrouce**_location_on_this_service

Reply:
nice catch 🤣